### PR TITLE
Create pt-br localizable.string

### PR DESCRIPTION
**Pull request by gloffreda**

### CODE REVIEW

**Comment by jbrophy17** [comment]
Thanks for the PR! Once I verify these translations, I'll be happy to merge this in

File: source/UberRides/Resources/pt-br.lproj/Localizable.string [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+"Your app is not authorized for the requested scopes." = "Seu aplicativo não está autorizado a fazer chamadas neste escopo.";
+"The Redirect URI provided did not match what was expected." = "A URI fornecida não corresponde ao esperado.";
+"A server error occurred." = "Erro inesperado de servidor.";
+"Unable to present the login view." = "Impossível de apresentar a tela de login.";
+"Unable to save the access token." = "Impossível de salvar token de acesso.";
+"Login is temporarily unavailable." = "Login temporariamente indisponível.";
+"User cancelled the login process." = "Usuário cancelou processo de login.";
+"Try Again" = "Tente novamente";
+"Cancel" = "Cancelar";
+"The operation you are attempting is not supported on the current device." = "A operação não pode ser executado neste tipo de equipamento.";
```
Comment (jbrophy17) [inline source/UberRides/Resources/pt-br.lproj/Localizable.string]: Could we change "equipamento" to "aparelho"?

File: source/UberRides/Resources/pt-br.lproj/Localizable.string [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+"%d min away" = "%d min longe";
+"%d mins away" = "%d mins longe";
```
Comment (jbrophy17) [inline source/UberRides/Resources/pt-br.lproj/Localizable.string]: On both of these, could we change to "a %d mins" instead of "%d mins longe" ?

Comment (gloffreda) [inline source/UberRides/Resources/pt-br.lproj/Localizable.string]: yes you can. We had bad time trying to fit a better world here. The problem is the length. 
But I think just "mins" is comprehensible.

**Comment by Sml995** [comment]
How can I change the localization?

File: source/UberRides/Resources/pt-br.lproj/Localizable.string (hunk 2 above)
Comment (gloffreda) [inline source/UberRides/Resources/pt-br.lproj/Localizable.string]: Do you still planning to add pt-br ?

**Comment by gloffreda** [comment]
Is there anything I can do to have this working ?

**Comment by edjiang** [comment]
@gloffreda I just took over this project; will see if there's anything else we need to do for localization. ^^ will see if there are any other next steps

**Comment by edjiang** [comment]
@gloffreda merged into the `swift-3-dev` branch. See #167 for how to install it.

Thanks so much for contributing!

**Comment by gloffreda** [comment]
May I suggest some improvements ? Just updated my project and localization works great. 
I just saw some sentences that can be improved.

**Comment by edjiang** [comment]
@gloffreda sure, no problem. Just submit another PR! :)

**Pull request merged**